presidio-image-anonymizer/presidio_image_anonymizer/image_anonymizer_engine.py

```diff
@@ -1,13 +1,15 @@
-from PIL import Image, ImageDraw
 from typing import Union, Tuple
+
+from PIL import Image, ImageDraw
+
 from presidio_image_anonymizer.image_analyzer_engine import ImageAnalyzerEngine
 
 
 class ImageAnonymizerEngine:
     """ImageAnonymizerEngine class only supporting redaction currently."""
 
     def anonymize(
-        self, image: object, fill: Union[int, Tuple[int, int, int]] = (0, 0, 0)
+            self, image: Image, fill: Union[int, Tuple[int, int, int]] = (0, 0, 0)

```

Comment:
nit: change image docstring to PIL Image

Reply:
Done :)

Reply:
Done.